<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faceManageWidget</class>
 <widget class="QWidget" name="faceManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
border-image: url(:/images/images/passBg2.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_title">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frame_title
{
	border-top-left-radius:3px;
	border-top-right-radius:3px;
	background-color:qlineargradient(x1:0,y1:0,x2:1,y2:0,stop:0 rgba(0,83,207,0.9),stop:1 rgba(66,183,255,0.9));
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_title">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_left">
           <property name="minimumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_left">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_logo">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/images/images/logo1.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_title">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>8</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>办公终端人脸识别</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frame_right">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/images/topHang.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_right">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_return">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_return">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton#btn_return
{
	border-image: url(:/images/images/return2.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_min">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_min
{
	border-image: url(:/images/images/min.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_close
{
	border-image: url(:/images/images/close.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid rgb(28, 95, 245);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#btn_search{
	color: rgb(75, 131, 255);
	background-color: rgb(233, 240, 255);
	border: 1px solid rgb(28, 95, 245);
	border-radius: 4px;
}

QPushButton#btn_search:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}

QPushButton#btn_search:pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}</string>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_face">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#btn_face{
	color: rgb(75, 131, 255);
	background-color: rgb(233, 240, 255);
	border: 1px solid rgb(28, 95, 245);
	border-radius: 4px;
}

QPushButton#btn_face:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}

QPushButton#btn_face:pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}</string>
           </property>
           <property name="text">
            <string>录入人脸</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>310</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>310</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>540</width>
             <height>295</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>540</width>
             <height>295</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollArea#scrollArea{
    border: 1px solid rgba(212, 229, 248, 1);
    background-color: rgb(255, 255, 255);
    border-radius: 4px;
}
QScrollArea#scrollArea QScrollBar:vertical {
    background: #f5f5f5;
    width: 10px;
    margin: 2px 0;
    border: none;
    border-radius: 5px;
}

QScrollArea#scrollArea QScrollBar::handle:vertical {
    background: #c5c5c5;
    min-height: 20px;
    border-radius: 5px;
}

QScrollArea#scrollArea QScrollBar::add-page:vertical,
QScrollArea#scrollArea QScrollBar::sub-page:vertical {
    background: #f5f5f5;
}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>538</width>
              <height>293</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
